docs/products/clickhouse/howto/fetch-query-statistics.rst

```diff
@@ -0,0 +1,24 @@
+Fetch query statistics for Aiven for ClickHouse®
+================================================
+Usually, query statistics in ClickHouse can be obtained using the system.query_log table, which stores statistics of each executed query, including memory usage and duration. In Aiven for ClickHouse®, the system.query_log table is currently not accessible for the purpose of obtaining query statistics.
```

Comment:
1. Please include a blank line below the article title.
2. Also, please include a section title above the first paragraph:
```
About fetching query statistics
-------------------------------
```
3. Please put all instances of "system.query_log" into `` so that they use the code font.